Formula/openiked.rb

```diff
@@ -7,6 +7,11 @@ class Openiked < Formula
   license "ISC"
   revision 1
 
+  livecheck do
+    url "https://cdn.openbsd.org/pub/OpenBSD/OpenIKED/"
+    regex(/href=.*?openiked-(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?openiked[._-]v?(\d+(?:\.\d+)+)\.t/i)
```
This is for consistency with other livecheck blocks and to add just a small amount of flexibility to the regex as well.